### DISCUSSION TITLE
find the sus among us impostor

### DISCUSSION THREAD

**Discussion by ItsEnderium** [CluckCluckChicken/Magnifier#58] [Thyroxine-binding globulin]
find the sus among us impostor
thats the name of this thread
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ඞ@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

hint: 👩‍🚀⛽🚉🚡📺🖥️🏧🥽⚙️📓

**Comment by JackyBorderCollie** [comment]
ok it is ⛽ because it looks very sussy

**Reply by ItsEnderium** [reply]
no i thought it was 📺

**Reply by JackyBorderCollie** [reply]
⛽ looks very sussy

**Comment by pikachub2005** [comment]
What about [this](https://github.com/CluckCluckChicken/Magnifier/blob/6eace593acb2910886a2fbf7f152c55325679311/Spyglass/wwwroot/images/yeet_goat.png)
